linera-service/tests/end_to_end_tests.rs

```diff
@@ -1127,3 +1147,78 @@ async fn test_end_to_end_social_user_pub_sub() {
     node_service1.assert_is_running();
     node_service2.assert_is_running();
 }
+
+#[test_log::test(tokio::test)]
+async fn test_end_to_end_fungible() {
+    use fungible::{AccountOwner, FungibleInitial};
+    use linera_base::{crypto::CryptoHash, data_types::Amount};
+    use std::collections::BTreeMap;
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+
+    let network = Network::Grpc;
+    let mut runner = TestRunner::new(network, 4);
+    let client = runner.make_client(network);
+
+    runner.generate_initial_validator_config().await;
+    client.create_genesis_config().await;
+    let chain = client.get_wallet().default_chain().unwrap();
+    runner.run_local_net().await;
+    let (contract, service) = runner.build_application("fungible2").await;
+
+    // The players
+    let owner_a = client.get_owner().expect("Failed to get the owner");
+    let account_owner_a = AccountOwner::User(owner_a);
+    let owner_b = Owner(CryptoHash::from([23, 22, 145, 21]));
```

Comment:
Let's use the real owner here too.

Reply:
Yeah, that becomes an obstacle because I cannot make the several chains work. I will have to see tomorrow what to do about it. Would appreciate advice about it.

Reply:
Thanks for your help.